samples/Gesture_APDS-9960/app/application.cpp

```diff
@@ -38,11 +38,11 @@ void IRAM_ATTR handleGesture()
 	}
 }
 
-void IRAM_ATTR interruptRoutine()
```

Comment:
Without the IRAM_ATTR the function will be stored in Flash. If that function is called during interrupt is there  a guarantee that it would be always possible to load the function machine code from flash?

Reply:
With the modifications all `InterruptDelegate` callbacks are handled via the task queue, in the same context as any other regular code, so there's no special handling required.

Reply:
>  I've attempted to explain this in the Basic_Interrupts sample.

Thanks, that is exactly what I was looking for :)